sips/sip-16/sip-016-token-metadata.md

```diff
@@ -0,0 +1,394 @@
+# Preamble
+
+SIP Number: 016
+
+Title: Schema Definition for Metadata for Digital Assets
+
+Author: Friedger Müffke (mail@friedger.de), Dan Trevino (dantrevino@gmail.com)
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Accepted
+
+Created: 7 November 2021
+
+License: CC0-1.0
+
+Sign-off:
+
+Layer: Traits
+
+# Abstract
+
+Non-fungible tokens - NFTs for short - are digital assets registered on blockchain with unique identifiers and properties that distinguish them from each other. SIP-009 defines the trait for how ownership of an NFT is managed. Fungible tokens - FTs for short - are digital assets where each token can be replaced by another token (see SIP-010). Semi-fungible tokens are digital assets where each token has a unique identifier and is dividable into fungible parts (see SIP-013). This SIP aims to provide a flexible standard to attach metadata to NFTs, like descriptions or urls to digital files. The same standard is applicable to fungible tokens.
```

Comment:
Small formatting nit -- can you make it so that lines are 80 columns?  It will make managing patches a lot less conflict-prone.  Thanks!